v2/internal/runner/runner.go

```diff
@@ -138,10 +138,12 @@ func New(options *types.Options) (*Runner, error) {
 	dupeCount := 0
 
 	// Handle single target
-	if options.Target != "" {
-		runner.inputCount++
-		// nolint:errcheck // ignoring error
-		runner.hostMap.Set(options.Target, nil)
+	if len(options.Target) != 0 {
+		for _, target := range options.Target {
```

Comment:
the "options.Target" should be renamed to "options.Targets", to reflect that it holds multiple values

Reply:
I assume checking for empty strings within the slice is still required.

Reply:
Does the CLI flag remain `-u` and `--target` ?

Reply:
Also, `options.Targets` already exists and takes a `filepath` for loading from file. I original wanted to rename this to `option.Targets` and rename `option.Targets` to `option.TargetFile` but that was request to change

Reply:
The CLI flag will definitely remain as is. In the options, the "Targets" could be renamed to TargetsPath or TargetsFilePath, because it contains a file path, and the current "Target" then could be renamed to "Targets".